app/models/user.rb

```diff
@@ -520,6 +520,14 @@ def editor_or_teacher_in?(lecture)
     in?(lecture.editors) || self == lecture.teacher
   end
 
+  def tutorials(lecture)
+    given_tutorials.where(lecture: lecture)
+  end
+
+  def has_tutorials?(lecture)
+    given_tutorials.where(lecture: lecture).size.positive?
```

Comment:
I think this could be expressed in a simpler way: `!given_tutorials.where(lecture:lecture).empty?`